util/extract.sh.tpl

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+%{load_statement}
+
+id=$(docker run -d --privileged %{flags} %{image} %{commands})
+
+docker wait $id
+docker cp $id:%{extract_file} .
```

Comment:
Should you pass in the output path instead of using "."?

Reply:
Hmm, which output path are you referring to? This should (?) guarantee that the file is always copied out to the bazel runfiles directory.